addons/mail/models/ir_attachment.py

```diff
@@ -76,6 +76,11 @@ def _delete_and_notify(self, message=None):
             )
         self.unlink()
 
+    def _field_store_repr(self, field_name):
+        if field_name == "as_author_access_token":
+            return [Store.Attr("as_author_access_token", lambda a: a._get_author_access_token())]
+        return [field_name]
```

Comment:
I'm now deprecating `_field_store_repr` too.
If repeated in several places, the internal code can simply be moved to a separate function (`_get_store_as_author_access_token_fields` or whatever the name) to be called where necessary.